libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/AppLockView/SFSDKBiometricViewController.m

```diff
@@ -65,10 +71,9 @@ - (void)loadView {
     self.iconView.accessibilityIdentifier = @"iconView";
     
     CAShapeLayer *iconCircle = [CAShapeLayer layer];
+    self.iconCircle = iconCircle;
     [iconCircle setPath:[UIBezierPath bezierPathWithRoundedRect:self.iconView.bounds cornerRadius:kSFIconCircleDiameter].CGPath];
-    iconCircle.strokeColor = self.viewConfig.borderColor.CGColor;
-    iconCircle.borderColor = self.viewConfig.borderColor.CGColor;
-    iconCircle.fillColor = self.viewConfig.secondaryColor.CGColor;
```

Comment:
CGColors aren’t dynamic, breaking them out so they can be updated on appearance change